ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -483,33 +495,40 @@ export default defineComponent({
     const { getValidation, MhrSectVal, MhrCompVal } = useMhrValidations(toRefs(getMhrRegistrationValidationModel.value))
     const { isValidDeceasedOwnerGroup } = useMhrInfoValidation(getMhrInfoValidation.value)
 
-    const localState = reactive({
-      currentlyEditingHomeOwnerId: -1,
-      reviewed: false,
-      showOwnerChangesDialog: false,
-      showSuffixError: false,
-      ownerToDecease: null as MhrRegistrationHomeOwnerIF,
-      isEditingMode: computed((): boolean => localState.currentlyEditingHomeOwnerId >= 0),
-      isAddingMode: computed((): boolean => props.isAdding),
-      showTableError: computed((): boolean => {
-        // For certain Transfers, we only need to check for global changes and do not show table error in other cases
-        if (isTransferToExecutorProbateWill.value ||
+    const showTableError = computed((): boolean => {
```

Comment:
I'm curious why this declared outside the localState and also before the localState?

Reply:
It is preferable to keep within the localState i think for a few reasons...

- For consistency with the majority of other components
- Returned with the localState for template access, without an additional return. 
- Not necessarily applicable in this scenario (since its already a computed value) but we want to define local properties within the `reactive()` method from composition api. It is the recommended approach.

The jist of that, the localState is essentially a proxy. This gives the component the means to mutate properties, objects or whatever values locally without affecting/relying on the source of those properties and also generating reactive hooks for the local properties. 

Can be confusing at first for sure, let me know if you want to discuss more!

Reply:
Changed back to be under localState